### PR TITLE
US002-updated tests and links to other pages

### PR DESCRIPTION
**Pull request by RyanCunning1**
Updated tests to correct method being used and passing, also links to training page

### CODE REVIEW

File: application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java [hunk 1]
```diff
@@ -382,6 +383,44 @@ public static JSONArray getJobFamilyUsingCapabilityID(String capabilityID){
...
+
+        ResultSet preJsonResultSet = null;
+
+        if (conn == null) {
+            RolesSpecJSON.put("Error no connection created");
+            return ResponseEntity.ok(RolesSpecJSON.toString());
+        }
+
+        try {
+
```
Comment (John-Neill-Kainos) [inline application/apiService/src/main/java/com/kainos/ea/apiService/controllers/apiService.java:399]: blank space after try, could you remove it please

File: application/apiService/src/test/java/com/kainos/ea/apiService/controllers/apiServiceTest.java [hunk 2]
```diff
@@ -91,7 +159,7 @@ void getCapabilityLeadDetailsUsingValidCapabilityName_ExpectJSONDataInResponseEn
     }
 
     @Test
-    void getCapabilityLeadDetailsUsingInvalidCapabilityName_ExpectEmptyJSONArrayInResponseEntity(){
+    void getCapabilityLeadDetailsUsingInvalidCapabilityName_ExpectEmptyJSONArrayInResponseEntity() {
```
Comment (John-Neill-Kainos) [inline application/apiService/src/test/java/com/kainos/ea/apiService/controllers/apiServiceTest.java:162]: all these tests namr have a space after the (), is there a reason for that ? If there isnt then could remove ?

File: routes.js [hunk 3]
```diff
@@ -127,4 +142,5 @@ router.get('/roles/:bandName/training', async function (req, res) {
         console.log(ex);
        }
 });
-    
+
+
```
Comment (John-Neill-Kainos) [inline routes.js:146]: Just again remove extra spaces

File: Views/UITests/src/main/java/pages/RolesPage.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    public String tableResponsibilityCheck(){
+        return tableResponsibility.getText();
+    }
+
+    public String tableJobSpecURLCheck(){
+        return tableJobSpecURL.getText();
+    }
+
+
```
Comment (liammc98) [inline Views/UITests/src/main/java/pages/RolesPage.java:43]: Remove the whitespace please

**Pull request closed without merging**